controllers/remote/suite_test.go

```diff
@@ -41,11 +41,11 @@ var (
 	ctx     = context.Background()
 )
 
-func TestAPIs(t *testing.T) {
+func TestGinkgoSuite(t *testing.T) {
```

Comment:
Not sure where `TestAPIs` came from but I thought this was more clear as to what is happening.

Reply:
Kubebuilder used to (probably still does) generate it like this